tests/components/axis/test_config_flow.py

```diff
@@ -43,7 +49,7 @@ async def test_flow_manual_configuration(hass):
             },
         )
 
-    assert result["type"] == "create_entry"
+    assert result["type"] == RESULT_TYPE_CREATE_ENTRY
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)